frontend/src/components/landingPage/ResultPart.vue

```diff
@@ -0,0 +1,277 @@
+<template>

```

Comment:
Why is this page part of the pull request? 
The feature is not yet complete. Please only include completed features.

Reply:
It has been removed now which is not used in this version.